wolfcrypt/src/rsa.c

```diff
@@ -3517,6 +3533,18 @@ static int RsaPrivateDecryptEx(const byte* in, word32 inLen, byte* out,
             }
             return ret;
         }
+    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+           #ifdef WOLF_CRYPTO_CB
+            if (key->devId != INVALID_DEVID) {
```

Comment:
Our indentation level should be 4 spaces.  It looks like you only have one space indent for the block inside `#ifdef WOLF_CRYPTO_CB`.

Reply:
Thanks! Fixed.